lm_eval/utils.py

```diff
@@ -60,6 +80,31 @@ def __iter__(self):
                     "input_len": outputs.attention_mask[sample].sum(),
                 }
 
+    def _make_infill_prompt(self, prefix, suffix):
+        """Make a prompt for infilling.
+        Currently supported only for official InCoder and SantaCoder implementations.
+        """
+        model_id = self.tokenizer.name_or_path
+        if model_id in ["facebook/incoder-1B", "facebook/incoder-6B"]:
+            self.tokenizer.add_special_tokens({"pad_token": "<pad>"})
+            return f"{prefix}<|mask:0|>{suffix}<|mask:0|>"
```

Comment:
it seems the DS1000 files uses `prefix + "<|mask:0|>\n" + suffix + "\n<|mask:0|>"` so we should make sure these newlines are added in suffix when it's returned by `get_prompt`

Reply:
Yes, have made this explicit now. Resolved [here](https://github.com/bigcode-project/bigcode-evaluation-harness/pull/39/commits/111d46fcb7006ffb6bec813a9b3125193ad2ca91)